client/components/status/status.test.js

```diff
@@ -456,6 +456,31 @@ describe("<Status /> interactions", () => {
     await tick();
     expect(spyFn.mock.calls.length).toBe(0);
   });
+  it("should submit loginFormRef after login", async () => {
+    validateToken.mockReturnValue(true);
+    props = createTestProps();
+    props.location.search = "";
+    props.userData = {
+      response_code: "AUTH_TOKEN_VALIDATION_SUCCESSFUL",
+      radius_user_token: "o6AQLY0aQjD3yuihRKLknTn8krcQwuy2Av6MCsFB",
+      username: "tester",
+      email: "tester@tester.com",
+      is_active: true,
+      is_verified: true,
+      phone_number: "+237672279436",
+    };
+    expect(true).toBe(true);
+    sessionStorage.setItem(`${props.orgSlug}_just_authenticated`, true);
+    wrapper = shallow(<Status {...props} />, {
+      context: {setLoading: jest.fn()},
+      disableLifecycleMethods: true,
+    });
+    const spyFn = jest.fn();
+    wrapper.instance().loginFormRef.current = {submit: spyFn};
+    wrapper.instance().componentDidMount();
+    await tick();
+    expect(spyFn.mock.calls.length).toBe(1);
```

Comment:
this test doesn't fail without the rest of the changes in this commit. Did you know this?

We must ensure tests fail with a clear message without the changes we introduce. Tests are the canary in the coal mine, please ensure the canary will save a human's life or their death will be pointless :joy: 

Therefore, please make sure:

1. the test fails without your changes
2. the test fails if you break the captive portal login feature on purpose by removing key parts and ensuring with manual testing the feature is broken

Without these 2 points, this code is pointless and we're just adding bloat to an already bloated codebase, please keep these suggestions in mind for any future tests you will be writing.